File: btjs_page/btjs/mod/checkers/in_range.js


 
/*
Function to identify if the input 
number is contained within the 
sending range.
*/
export function in_range(number, range_arr) {
    if((number >= range_arr[0])
    && (number <= range_arr[1])) {
        return true;
    }   
    return false;
}

File: compiler/src/btpy_lib/btpy/src/btpy/btpy_checkers/mod/in_range/in_range.py




def in_range(number: int, range_arr: list)\
        -> bool:
    """
    Function to identify if the input 
    number is contained within the 
    sending range.
    """
    if(not type(number) == int
    and not type(number) == float):
        raise Exception(
            f"is not a number {number}"
        )
    if(number >= range_arr[0]
    and number <= range_arr[1]):
        return True
    return False
